rustup.yaml

```diff
@@ -29,6 +29,13 @@ pipeline:
       tag: ${{package.version}}
       expected-commit: 54dd3d00fd20e64dc522517a0d7be4285570a2f1
 
+  - name: Update package with cargo
```

Comment:
Why aren't we changing idna version to a non-vulnerable v1.0.0?

Reply:
I'd have expected that we must change that version instead of re-installing openssl/openssl-sys here.